src/pages/workspace/WorkspaceNewRoomPage.js

```diff
@@ -177,6 +181,7 @@ function WorkspaceNewRoomPage(props) {
                         >
                             <View style={styles.mb5}>
                                 <RoomNameInput
+                                    ref={(el) => (roomNameInputRef.current = el)}
                                     inputID="roomName"
                                     isFocused={props.isFocused}
                                     shouldDelayFocus
```

Comment:
Do we still need this? `shouldDelayFocus` and `autoFocus`

Reply:
No, I don't think we do. I'll remove this.

Reply:
@abdulrahuman5196 Actually, after testing, these are still needed on web for cases when the user refreshes the browser window while the Room tab is active.